metadata_test.go

```diff
@@ -0,0 +1,214 @@
+package acceptance_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/layout"
+	. "github.com/onsi/gomega"
+	"github.com/paketo-buildpacks/packit/v2/vacation"
+	"github.com/sclevine/spec"
+
+	. "github.com/paketo-buildpacks/jam/integration/matchers"
+	. "github.com/paketo-buildpacks/packit/v2/matchers"
+)
+
+func testMetadata(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect = NewWithT(t).Expect
+
+		tmpDir string
+	)
+
+	it.Before(func() {
+		var err error
+		tmpDir, err = os.MkdirTemp("", "")
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	it.After(func() {
+		Expect(os.RemoveAll(tmpDir)).To(Succeed())
+	})
+
+	it("builds base stack", func() {
+		var buildReleaseDate, runReleaseDate time.Time
+
+		by("confirming that the build image is correct", func() {
+			dir := filepath.Join(tmpDir, "build-index")
+			err := os.Mkdir(dir, os.ModePerm)
+			Expect(err).NotTo(HaveOccurred())
+
+			archive, err := os.Open(stack.BuildArchive)
+			Expect(err).NotTo(HaveOccurred())
+			defer archive.Close()
+
+			err = vacation.NewArchive(archive).Decompress(dir)
+			Expect(err).NotTo(HaveOccurred())
+
+			path, err := layout.FromPath(dir)
+			Expect(err).NotTo(HaveOccurred())
+
+			index, err := path.ImageIndex()
+			Expect(err).NotTo(HaveOccurred())
+
+			indexManifest, err := index.IndexManifest()
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(indexManifest.Manifests).To(HaveLen(1))
+			Expect(indexManifest.Manifests[0].Platform).To(Equal(&v1.Platform{
+				OS:           "linux",
+				Architecture: "amd64",
+			}))
+
+			image, err := index.Image(indexManifest.Manifests[0].Digest)
+			Expect(err).NotTo(HaveOccurred())
+
+			file, err := image.ConfigFile()
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(file.Config.Labels).To(SatisfyAll(
+				HaveKeyWithValue("io.buildpacks.stack.id", "io.buildpacks.stacks.jammy"),
```

Comment:
Shouldn't this label have the value `io.buildpacks.stacks.jammy.base` ?

Reply:
The [RFC outlines](https://github.com/paketo-buildpacks/rfcs/blob/main/text/stacks/0004-jammy-jellyfish.md#stack-ids) that the stack ID for base and full will be the same, `io.buildpacks.stacks.jammy`. This aligns with what is currently done in Bionic.

Reply:
According to the RFC `io.buildpacks.stacks.jammy` is correct  https://github.com/paketo-buildpacks/rfcs/blob/ba9273ded6c65274276677b1ad5db5e61029060d/text/stacks/0004-jammy-jellyfish.md#stack-ids

This is also the case for bionic stacks:
- full id: `io.buildpacks.stacks.bionic`
- base id: `io.buildpacks.stacks.bionic`
- tiny id: `io.paketo.stacks.tiny`

Reply:
yes, sorry, I was looking at `tiny` and forgetting that we treat it differently than `base` and `full`.